clients/cli/src/main/java/org/apache/gravitino/cli/ErrorMessages.java

```diff
@@ -29,7 +29,7 @@ public class ErrorMessages {
   public static final String UNKNOWN_SCHEMA = "Unknown schema name.";
   public static final String UNKNOWN_TABLE = "Unknown table name.";
   public static final String MALFORMED_NAME = "Malformed entity name.";
-  public static final String MISSING_NAME = "Missing name.";
+  public static final String MISSING_NAME = "Missing --name option.";
```

Comment:
Will "--" be redundant if saying it is an "option"?

Reply:
Perhaps, don't really care either way, was just making it explicit for the user,

Reply:
Ok, get it.